hack/ci/yake-local/garden-content/controller-registrations.yaml

```diff
@@ -5,9 +5,9 @@ metadata:
   values:
-    image: ghcr.io/yakecloud/gardener-extension-provider-local:v1.93.0
+    image: ghcr.io/yakecloud/gardener-extension-provider-local:1.94.0
```

Comment:
provider-local has not been build yet, we need to change when that is triggered somehow